processor_test.go

```diff
@@ -716,10 +717,14 @@ func TestProcessor_Serialization(t *testing.T) {
 		State[MyAppContext, MyOverallContext, MyJobContext]{
 			TriggerState: TRIGGER_STATE_NEW,
 			Exec: func(ctx context.Context, ac MyAppContext, oc MyOverallContext, jc MyJobContext) (MyJobContext, string, []KickRequest[MyJobContext], error) {
+				if jc.Count == 1 {
+					return jc, STATE_DONE, nil, errors.New("errored again")
```

Comment:
Why errors again?

Reply:
to confirm that if multiple errors are detected, they are recorded separately and don't clobber each other. You can see this assertion lower in the test case